hera_mc/geo_location.py

```diff
@@ -69,9 +70,12 @@ class GeoLocation(MCDeclarativeBase):
     elevation = Column(Float)
     "Elevation in m"
 
-    created_date = NotNull(DateTime(timezone=True))
+    created_gpstime = NotNull(BigInteger)
     "The date when the station assigned by project."
 
+    def gps2Time(self):
```

Comment:
If this method is just making a new convenience attribute on the class, why not call it in the __init__ method? You can write a custom __init__ that calls the parent class __init__ like this `super(UVData, self).__init__()` and then add anything you'd like for your custom class.